app/workers/kyc/kycaid/document_worker.rb

```diff
@@ -76,10 +76,11 @@ def selfie_image_params(selfie_image)
       end
 
       def verification_params
+        schema = Barong::App.config.tls_enabled ? 'https' : 'http'
         {
           applicant_id: @applicant_id,
           types: ['DOCUMENT', 'FACIAL'],
-          callback_url: "#{Barong::App.config.domain}/api/v2/barong/public/kyc"
+          callback_url: "#{schema}://#{Barong::App.config.domain}/api/v2/barong/public/kyc"
```

Comment:
We come with a little improvement
It seems it will be like a Breaking Change for our users
So maybe lets do like this:
Check if #{Barong::App.config.domain} has https/http schema, you should get only domain from this URI, if there is no uri put as it is